### DISCUSSION TITLE
Question: how to load a local file in electron without 'loadBlob' in v7 ?

### DISCUSSION THREAD

**Discussion by matt-42** [katspaugh/wavesurfer.js#2924] [General]
`loadBlob` has been removed in v7 and I was using it to load music files in v6. Could you provide example showing how we can load audio files in electron with v7 without loadBlob ?

**Comment by katspaugh** [comment]
You could create a URL object from your blob and pass that to wavesurfer:

```
const url = URL.createObjectURL(blob)
wavesurfer.load(url)
```

See https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL_static
